packages/core-data/src/resolvers.js

```diff
@@ -126,14 +126,13 @@ export function* hasUploadPermissions() {
  * @param {Object} post The post that is the parent of the autosave
  */
 export function* getAutosave( post ) {
-	const {
-		id,
-		type,
-	} = post;
+	const { id, type } = post;
 
-	const postTypePathPart = type === 'post' ? 'posts' : type;
+	const postTypeEntities = yield getKindEntities( 'postType' );
+	const entity = find( postTypeEntities, { name: type } );
+	const { baseURL } = entity;
```

Comment:
I have no idea if this is the right way to get the correct `baseURL` for the post type. Some validation or advice on alternatives would be appreciated.